static/javascripts/app.js

```diff
@@ -107,6 +107,23 @@ function changeQuality(event, ui) {
   lowPassFilterNode.Q.value = ui.value * 30;
 }
 
+
+$(window).keypress(function (e) {
+  if (e.keyCode === 0 || e.keyCode === 32) {
+    e.preventDefault();
+    var $span = $('#play-pause').children("span");
```

Comment:
the following lines are duplicate of what is written in playPauseListener().
It would be nice to create a function and map it to the click and space bar in another function.